INFO     root:test_02_usersync.py:83 ==========开始执行用例：usersync_syncConfig_list_01==========
INFO     root:runcase.py:30 -·-·-·-·-·-·-·-·-·-发送请求并接受信息 START-·-·-·-·-·-·-·-·-·-
INFO     root:handleyaml.py:94 Yaml测试用例基础架构检查通过
INFO     root:runcase.py:47 重组后的用例信息：{'name': '同步配置列表搜索-无搜索条件', 'description': '不输入搜索条件，点击搜索，查看数据源列表', 'request': {'data': {'pageNum': 1, 'pageSize': 20}, 'method': 'POST', 'address': '/api/usersync/v1/syncConfig', 'headers': {'Content-Type': 'application/json', 'token': 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJhdWQiOiJhcHBsaWNhdGlvbiIsInN1YiI6IjdlYTYwMzc0LTdjZTQtNGI0YS05ODNmLTYyMDNiZjQ1NTUxMSIsImlzcyI6ImJtc19zZXJ2ZXIiLCJzZXNzaW9uSWQiOiI3ZWE2MDM3NC03Y2U0LTRiNGEtOTgzZi02MjAzYmY0NTU1MTEiLCJleHAiOjE2ODc5NTMzMDUsImlhdCI6MTY4Nzc3MzMwNX0.ZEh_kobQplJq6Ir4jExaWcfxbJsVc9q3auT0UTeNUCqHMgPwJ8DcBIWCU3ymzQEDJAzxG6-6_IMLyGTayZfRDQ', 'Referer': 'https://192.168.148.174:31000/sub-app-unity/userSync'}}, 'validate': {'code': 200, 'response': {'statusCode': 0, 'msg': 'success'}}, 'epic': '用户管理', 'feature': '用户同步管理-用户同步', 'story': '同步配置列表搜索接口', 'base_url': 'https://192.168.148.174:31000', 'caseid': 'usersync_syncConfig_list_01'}
INFO     root:runcase.py:83 请求地址：https://192.168.148.174:31000/api/usersync/v1/syncConfig
INFO     root:runcase.py:84 请求方法：POST
INFO     root:runcase.py:85 请求头：{'Content-Type': 'application/json', 'token': 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJhdWQiOiJhcHBsaWNhdGlvbiIsInN1YiI6IjdlYTYwMzc0LTdjZTQtNGI0YS05ODNmLTYyMDNiZjQ1NTUxMSIsImlzcyI6ImJtc19zZXJ2ZXIiLCJzZXNzaW9uSWQiOiI3ZWE2MDM3NC03Y2U0LTRiNGEtOTgzZi02MjAzYmY0NTU1MTEiLCJleHAiOjE2ODc5NTMzMDUsImlhdCI6MTY4Nzc3MzMwNX0.ZEh_kobQplJq6Ir4jExaWcfxbJsVc9q3auT0UTeNUCqHMgPwJ8DcBIWCU3ymzQEDJAzxG6-6_IMLyGTayZfRDQ', 'Referer': 'https://192.168.148.174:31000/sub-app-unity/userSync'}
INFO     root:runcase.py:95 请求参数：{'pageNum': 1, 'pageSize': 20}
DEBUG    urllib3.connectionpool:connectionpool.py:1003 Starting new HTTPS connection (1): 192.168.148.174:31000
DEBUG    urllib3.connectionpool:connectionpool.py:456 https://192.168.148.174:31000 "POST /api/usersync/v1/syncConfig HTTP/1.1" 200 284
INFO     root:runcase.py:157 请求接口结果： {'statusCode': 0, 'msg': 'success', 'data': {'totalCount': 1, 'list': [{'id': 26, 'modifyTime': '2023-06-26 17:58:07', 'userSourceId': '37', 'jobTime': '00 39 15 10 * ?', 'isIncrement': 1, 'threshold': 0, 'taskSavePeriod': 180, 'syncName': '自动化测试同步配置', 'syncType': '0', 'allowUpdate': True}]}}
INFO     root:checkresult.py:113 code断言通过, 期望状态码:200, 实际状态码:200
INFO     root:checkresult.py:48 返回结果断言通过, 期望返回值:{'statusCode': 0, 'msg': 'success'}, 实际返回值:{'statusCode': 0, 'msg': 'success', 'data': {'totalCount': 1, 'list': [{'id': 26, 'modifyTime': '2023-06-26 17:58:07', 'userSourceId': '37', 'jobTime': '00 39 15 10 * ?', 'isIncrement': 1, 'threshold': 0, 'taskSavePeriod': 180, 'syncName': '自动化测试同步配置', 'syncType': '0', 'allowUpdate': True}]}}
INFO     root:runcase.py:63 -·-·-·-·-·-·-·-·-·-发送请求并接受信息 END-·-·-·-·-·-·-·-·-·-